src/FluentAvalonia/UI/Controls/TabView/TabView.cs

```diff
@@ -131,7 +131,7 @@ protected override void OnApplyTemplate(TemplateAppliedEventArgs e)
 
         UpdateListViewItemContainerTransitions();
 
-        OnLoaded();
+        OnLoaded(e);
```

Comment:
This probably isn't correct passing a new TemplateAppliedEventArgs to the OnLoaded method. It only works since they are both RoutedEventArgs but logically it doesn't make sense to do this.

This might be a wider bug here as OnLoaded() should be invoked directly by the control base class itself as part of its lifetime. I wonder if this is a hack-fix for something.